lib/cluster.js

```diff
@@ -230,6 +231,9 @@ Cluster.prototype.parseStats = function parseStats(stats) {
         cluster = new Partition();
         cluster.membershipChecksum = stats.membershipChecksum;
         cluster.membership = stats.members;
+        cluster.aliveCount = _.filter(stats.members, {status: 'alive'}).length;
```

Comment:
Truth, but if you want to avoid a little ugliness, I think this'd work.

```
var statusGroups = _.groupBy(stats.members, function group(member) { return member.status; });
cluster.aliveCount = _.size(statusGroups.alive);
cluster.suspectCount = _.size(statusGroups.suspect);
cluster.faultyCount = _.size(statusGroups.faulty);
```